### PR TITLE
Fixed so the footer doesnt clip into the body and also added link to …

### PR DESCRIPTION
**Pull request by Kookiie88**
#### Feature Enhancements:
1. **Profile/Login Button:** Added a profile/login button to the header for improved navigation and user accessibility.
2. **Footer Adjustment:** Fixed the footer to ensure it no longer clips into the body, providing a cleaner and more consistent layout.

These updates enhance the overall user experience by improving navigation and ensuring a more polished interface.

### CODE REVIEW

File: kinogoteborg/src/app/components/headerFooter/header.jsx
Comment (MatsJoh50) [inline kinogoteborg/src/app/components/headerFooter/header.jsx:1]: future problem,
add hover: to links

and maybe import clsx to make future classchanges easier.

**Review by MatsJoh50 — APPROVED** [review]
overall good.
as commented in header.jsx,
might want to add clsx and hover: in the future

**Review by Weeezzt — APPROVED** [review]
What i can see the removal of the fixed made it much better and the addition of the log in / profile link in the header is a nice addition!

**Comment by Kookiie88** [comment]
This pull request is linked to PBI #18

**Pull request merged**